ui/src/containers/Client.js

```diff
@@ -56,6 +57,8 @@ class Client extends Component {
   }
 
   render() {
+    const { pathname } = this.props.location;
+    const clientId = pathname.substr(pathname.lastIndexOf('/') + 1);
```

Comment:
You can get clientId from `this.props.match.params.id`

Reply:
thanks